src/test/java/teammates/test/cases/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -160,6 +161,16 @@ private void testInviteInstructorAction() {
                 Const.StatusMessages.COURSE_REMINDER_SENT_TO + "InsCrsEdit.newInstr@gmail.tmt");
     }
 
+    private void testCancelAddingNewInstructor() {
+        ______TS("Cancelling adding of a new instructor");
```

Comment:
maybe you should not single out creating a new `add instructor form` from pressing the cancel button...after all each time before you press cancel you have to create a new form...

And I think you can hv one other test case, e.g. fill in the form and press cancel, or fill in half of the form and press cancel.